### PR TITLE
Branch testdelete

### PR DESCRIPTION
**Pull request by TeeRenJing**
Lets improve our test coverage to cover all prescription related code.

### CODE REVIEW

**Review by shyanyong — APPROVED** [review]
LGTM

File: src/test/java/seedu/address/logic/commands/DeletePrescriptionCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+                new seedu.address.model.UserPrefsPrescription());
+        expectedModel = new seedu.address.model.ModelManagerPrescription(model.getPrescriptionList(),
+                new seedu.address.model.UserPrefsPrescription());
+    }
+    @Test
+    public void execute_validIndexUnfilteredList_success() {
+        seedu.address.model.prescription.Prescription prescriptionToDelete = model.getFilteredPrescriptionList()
+                .get(INDEX_FIRST_PRESCRIPTION.getZeroBased());
+        DeletePrescriptionCommand deletePrescriptionCommand = new DeletePrescriptionCommand(INDEX_FIRST_PERSON);
+
```
Comment (RoeReRe) [inline src/test/java/seedu/address/logic/commands/DeletePrescriptionCommandTest.java:42]: Inconsistent imports.

Should remove all instances of `INDEX_XXX_PERSON` and change to `INDEX_XXX_PRESCRIPTION` as the Person variant will be deprecated.

File: src/test/java/seedu/address/logic/parser/DeletePrescriptionCommandParserTest.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                DeletePrescriptionCommand.MESSAGE_USAGE));
+    }
+
+    @Test
+    public void parse_emptyPreamble_failure() {
+        // No index specified
+        assertParseFailure(parser, "", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                DeletePrescriptionCommand.MESSAGE_USAGE));
+    }
+
```
Comment (RoeReRe) [inline src/test/java/seedu/address/logic/parser/DeletePrescriptionCommandParserTest.java:35]: Would also include whitespace preamble `" "` test case personally, but non issue.

File: src/test/java/seedu/address/logic/parser/DeletePrescriptionCommandParserTest.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    @Test
+    public void parse_validArgs_success() {
+        // Valid index
+        assertParseSuccess(parser, " i/1", new DeletePrescriptionCommand(Index.fromOneBased(1)));
+    }
+
+    @Test
+    public void parse_invalidArgs_failure() {
+        // Valid index
+        assertParseFailure(parser, " i/0", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
```
Comment (RoeReRe) [inline src/test/java/seedu/address/logic/parser/DeletePrescriptionCommandParserTest.java:25]: // **invalid** index

**Review by RoeReRe — COMMENTED** [review]
typos

**Review by ChongWeiJie29 — APPROVED** [review]
lgtm

**Pull request merged**